sdks/go/pkg/beam/io/filesystem/gcs/gcs_test.go

```diff
@@ -43,6 +44,21 @@ func TestGCS_FilesystemNew(t *testing.T) {
 }
 
 func TestGCS_direct(t *testing.T) {
+	testGCS_direct(t)
+}
+
+func TestGCS_BillingProjectHookEnable(t *testing.T) {
+	billingProject := "whatever"
+	RequesterBillingProject(billingProject)
+	_, err := hooks.RunInitHooks(context.Background())
+	if err != nil {
+		t.Errorf("error to init hooks = %v", err)
+	}
+	//TODO how assert project configured by hook?
```

Comment:
@lostluck how can i assert project set by hook?

Reply:
Call: hooks.IsEnabled. It will return if the hook is enabled, and the options that were set.

https://pkg.go.dev/github.com/apache/beam/sdks/v2@v2.64.0/go/pkg/beam/core/util/hooks#IsEnabled